src/type/definition.js

```diff
@@ -92,6 +110,15 @@ export type GraphQLOutputType =
     GraphQLList<GraphQLOutputType>
   >;
 
+export function assertOutputType(type: GraphQLType): GraphQLOutputType {
+  invariant(
+    isOutputType(type),
+    'GraphQL: Expected type `%s` to be a GraphQLOutputType',
+    type.name
```

Comment:
`String(type)` - not all types are named, like list and non-null.

graphql-js's `invariant` doesn't support printf format strings - so instead this should be:

```js
`Expected type "${String(type)}" to be an output type.`
```